utils/io.go

```diff
@@ -10,19 +10,22 @@ import (
 
 // WriteKeyToPemFile writes encode of key and purpose to the file
 func WriteKeyToPemFile(fileName string, keyPurpose string, key []byte) error {
```

Comment:
Can we extend the existing `io_test.go` to check that the file permissions are the expected ones?